deepmd/train/trainer.py

```diff
@@ -617,3 +643,10 @@ def get_evaluation_results(self, batch_list):
                     sum_results[k] = sum_results.get(k, 0.) + v * results["natoms"]
         avg_results = {k: v / sum_natoms for k, v in sum_results.items() if not k == "natoms"}
         return avg_results
+    
+    def save_compressed(self):
+        """
+        Save the compressed graph
+        """
+        if self.is_compress:
+            self.saver.save (self.sess, os.getcwd() + "/" + self.save_ckpt)
```

Comment:
Use `os.path.json` instead of hard-code `/`, to prevent that someday we suddenly want to support windows.